client/verta/verta/operations/monitoring/alert/_entities/alert.py

```diff
@@ -51,6 +51,12 @@ class Alert(entity._ModelDBEntity):
 
     """
 
+    _ONEOF_ALERTER_ATTR = {
+        _AlertService.AlerterTypeEnum.FIXED: "alerter_fixed",
+        _AlertService.AlerterTypeEnum.REFERENCE: "alerter_reference",
+        _AlertService.AlerterTypeEnum.RANGE: "alerter_range",
+    }
```

Comment:
Somewhat similar to the comment in _alerter.py: I think we could have these strings defined on our alerter classes to be used in `_create_proto_internal()` to avoiding maintain an explicit mapping.

(though maybe that'd also require catching an `AttributeError` from `getattr()`)